test/e2e/storage/testsuites/volume_modify.go

```diff
@@ -82,10 +85,6 @@ func (v *volumeModifyTestSuite) GetTestSuiteInfo() storageframework.TestSuiteInf
 }
 
 func (v *volumeModifyTestSuite) SkipUnsupportedTests(driver storageframework.TestDriver, pattern storageframework.TestPattern) {
-	_, ok := driver.(storageframework.VolumeAttributesClassTestDriver)
-	if !ok {
```

Comment:
Removed because now there are tests for CSI Drivers that DO NOT support ModifyVolume capability.

The init function that relies on `driver.(storageframework.VolumeAttributesClassTestDriver)` still has this check and skip.

Reply:
Question: Should ModifyVolume capability false tests NOT be in this test suite, and instead just have csimock tests that are made in future? Or are these additional tests still worth including here:

- "should not modify volume when csi driver does not support volume modification"
- "should not create volume with VAC when csi driver does not support volume modification"

Reply:
> Question: Should ModifyVolume capability false tests NOT be in this test suite

Sure, let's follow the same style like volumeExpandTestSuite

> Or are these additional tests still worth including here:

It's still worth adding e2e tests. We can add those with the csimock driver. If the driver does not support it, but the PVC is created or modified successfully, this is unreasonable and will cause misunderstanding to the user.

Reply:
The tests under storage/testsuites are mainly for CSI driver authors to test their driver implementation. IMO, these negative tests are mainly testing k8s and not the CSI driver implementation, so it's better to move it outside.

Reply:
Great, I will remove these two tests then from here.

I believe other test cases in PR (plus the one case we must temporarily leave out) would be useful to csi driver authors because one of them already caught a non-spec compliance bug in EBS CSI Driver...

Reply:
Sounds good to me!